capstone/src/main/java/com/google/sps/servlets/SmallOwnedRestaurantsDataServlet.java

```diff
@@ -100,4 +106,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
     response.sendRedirect("/admin.html");
   }
+
+  public void clearDatastore(){
+    Query restaurantQuery = new Query("Restaurant");
```

Comment:
We'll have to change the name once we integrate

Reply:
for sure